### DISCUSSION TITLE
How can I access the auth users table in my custom table?

### DISCUSSION THREAD

**Discussion by uzair11C** [supabase/supabase#28032] [Questions] [answered]
Hi there, I am a frontend developer, so I don't know much about databases, so I am using Supabse. I am trying to create an orders table that would reference the "users" table through which we manage the auth, but when I run the query to create an orders table, it gives me an error: 
`ERROR:  42P01: relation "users" does not exist` . Here's my query:

```CREATE TABLE orders (
  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
  user_id uuid REFERENCES users(id) ON DELETE CASCADE,
  product_id uuid REFERENCES products(id) ON DELETE CASCADE,
  quantity integer NOT NULL,
  total_price numeric NOT NULL,
  order_status text CHECK (order_status IN ('pending', 'completed', 'cancelled')) DEFAULT 'pending',
  created_at timestamp with time zone DEFAULT now()
);
```
How can I access that table?

**Comment by silentworks** [comment, marked as answer]
The Supabase docs recommend against this because if you need to query a relationship from the API you won't be able to access the `auth` schema as this would be a security issue. You should create a `profiles` table or something of that sort. 

If you want to continue the way you are going you just need to add the schema name to the table reference, so `auth.users(id)` instead of just `users(id)`.

How this should be handled: https://supabase.com/docs/guides/auth/managing-user-data

**Reply by uzair11C** [reply]
Thanks a lot, I didn't know about security issues. I will create a dedicated users table and use it's id.
